tasks/auth_initialization.yml

```diff
@@ -9,7 +9,7 @@
     name: "{{ item.name }}"
     password: "{{ item.password }}"
     roles: "{{ item.roles }}"
-    login_host: 127.0.0.1
+    login_host: "{{ mongodb_listen_iface }}"
```

Comment:
Well, I was thinking about to not confuse it with vars like `mongodb_login_ip`. But you're right - IP sounds much better for this purpose. I'll update the name.